backend/src/main/java/de/bund/digitalservice/ris/norms/application/exception/AnnouncementNotFoundException.java

```diff
@@ -0,0 +1,9 @@
+package de.bund.digitalservice.ris.norms.application.exception;
+
+/** Indicates that the requested norm does not exist. */
+public class AnnouncementNotFoundException extends RuntimeException {
+
+  public AnnouncementNotFoundException(final String eli) {
+    super("Announcement with eli %s does not exist".formatted(eli));
```

Comment:
Suggestion: Announcements themselves don't have ELIs, but they refer to a norm with an ELI, right? In that case the message should read something like "Announcement for norm with eli XY does not exist".

Reply:
Frankly, I did not know. Thanks!